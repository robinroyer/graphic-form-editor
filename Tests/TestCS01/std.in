R r01 10 10 20 20
R r02 0 0 20 20
R r03 5 5 30 30
S s01 0 0 20 20
LIST
DELETE s01
LIST
EXIT