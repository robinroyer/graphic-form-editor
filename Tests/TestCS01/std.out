OK
#New object: r01
OK
#New object: r02
OK
#New object: r03
OK
#New object: s01
R r01 10 10 20 20
R r02 0 0 20 20
R r03 5 5 30 30
OK
#Objects sucsessfully removed
R r03 5 5 30 30
